pygmt/datasets/earth_age.py

```diff
@@ -76,12 +76,8 @@ def load_earth_age(
     Note
     ----
     The registration and coordinate system type of the returned
-    :class:`xarray.DataArray` grid can be accessed via the GMT accessors (i.e.,
-    ``grid.gmt.registration`` and ``grid.gmt.gtype`` respectively). However, these
-    properties may be lost after specific grid operations (such as slicing) and will
-    need to be manually set before passing the grid to any PyGMT data processing or
-    plotting functions. Refer to :class:`pygmt.GMTDataArrayAccessor` for detailed
-    explanations and workarounds.
+    :class:`xarray.DataArray` grid can be accessed via the *gmt* accessor. Refer to
+    :class:`pygmt.GMTDataArrayAccessor` for detailed explanations and limitations.
```

Comment:
I've also simplified the notes in the load_xxx functions. Please review it and then I'll apply the changes to other load_xxx functions after approval.